src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -280,7 +291,10 @@ public final Response getFastTrackGameboard(@Context final Request request,
 
         try {
             GameboardDTO gameboard;
-
+            if (!Constants.FASTTRACK_GAMEBOARD_WHITELIST.contains(gameboardId)) {
```

Comment:
What was the motivation for the white list check? The endpoint doesn't look that different to the normal gameboard one.

Reply:
FastTrack progress is calculated using more question results than the standard gameboard.
I (ab)use tags to represent the collection and structure of these related questions so to get all the questions I'm interested in, I query for questions including the gameboard ID as a tag.
I white-list the gameboard ID to avoid querying elasticsearch with a string which hasn't been validated. It is not really a problem at the moment as it is only staff who have the ability to set gamebaord IDs.